### ISSUE TITLE
restarting loop

### ISSUE THREAD

**Issue by davidbdeath** [itzg/docker-minecraft-server#452]
loaded it up and it said restarting for 10 min before i deleted it and gave up.

**Comment by itzg** [comment]
If you happen to have the container logs, it will help to diagnose this. My only guess without seeing logs is that the health check may have timed out and was attempting to restart the container. You can extend the default health interval by adding something like `--health-start-period 5m` to the docker run.

**Comment by prusayn** [comment]
I had same problem.
Log:
```
[init] Setting initial memory to 1G and max to 1G
[init] Starting the Minecraft server...
[13:10:08] [main/FATAL]: Failed to start the minecraft server
joptsimple.UnrecognizedOptionException: noconsole is not a recognized option
	at joptsimple.OptionException.unrecognizedOption(OptionException.java:108) ~[minecraft_server.1.15.2.jar:?]
	at joptsimple.OptionParser.handleLongOptionToken(OptionParser.java:510) ~[minecraft_server.1.15.2.jar:?]
	at joptsimple.OptionParserState$2.handleArgument(OptionParserState.java:56) ~[minecraft_server.1.15.2.jar:?]
	at joptsimple.OptionParser.parse(OptionParser.java:396) ~[minecraft_server.1.15.2.jar:?]
	at net.minecraft.server.MinecraftServer.main(SourceFile:906) [minecraft_server.1.15.2.jar:?]
```

Deleting `CONSOLE: "false"` in docker-compose file fixed it.

**Comment by itzg** [comment]
@prusayn I'll look into that since it sounds like they removed that option at some point

**Comment by dannytsang** [comment]
I have encountered this too and removing `CONSOLE: "false"` also worked as per @prusayn .

For completeness, snippet from the log when it goes into a restart loop:

> minecraft    | [init] Resolved version given LATEST into 1.15.2
> minecraft    | [init] Resolving type given VANILLA
> minecraft    | [init] server.properties already created, skipping
> minecraft    | [init] log4j2.xml already created, skipping
> minecraft    | [init] Checking for JSON files.
> minecraft    | [init] Copying any mods over...
> minecraft    | [init] Setting initial memory to 1G and max to 1G
> minecraft    | [init] Starting the Minecraft server...
> minecraft    | [20:34:45] [main/FATAL]: Failed to start the minecraft server
> minecraft    | joptsimple.UnrecognizedOptionException: noconsole is not a recognized option
> minecraft    | 	at joptsimple.OptionException.unrecognizedOption(OptionException.java:108) ~[minecraft_server.1.15.2.jar:?]
> minecraft    | 	at joptsimple.OptionParser.handleLongOptionToken(OptionParser.java:510) ~[minecraft_server.1.15.2.jar:?]
> minecraft    | 	at joptsimple.OptionParserState$2.handleArgument(OptionParserState.java:56) ~[minecraft_server.1.15.2.jar:?]
> minecraft    | 	at joptsimple.OptionParser.parse(OptionParser.java:396) ~[minecraft_server.1.15.2.jar:?]
> minecraft    | 	at net.minecraft.server.MinecraftServer.main(SourceFile:906) [minecraft_server.1.15.2.jar:?]
> minecraft    | 2020-04-13T20:34:45.909Z	INFO	mc-server-runner	Done
> minecraft exited with code 0

**Comment by itzg** [comment]
The latest image pushed to Docker Hub will now ignore "CONSOLE=false" when the minecraft server version is 1.14 or newer.
